compiler/testData/codegen/box/initializers/initializers1.kt

```diff
@@ -3,7 +3,6 @@
  * that can be found in the LICENSE file.
  */
 // KT-66103: companion object is not initialized
```

Comment:
1. It is just a tiny, minor thing. This comment is related to the following line, so you can remove it, too.